RpcNep5Tracker/RpcNep5Tracker.cs

```diff
@@ -191,7 +191,7 @@ public void OnPersist(Snapshot snapshot, IReadOnlyList<Blockchain.ApplicationExe
                     script = sb.ToArray();
                 }
 
-                ApplicationEngine engine = ApplicationEngine.Run(script, snapshot);
+                ApplicationEngine engine = ApplicationEngine.Run(script, snapshot,testMode:true);
```

Comment:
Don't use `testMode` because it can't prevent infinite loop attacks.

Reply:
Got it.

Reply:
NOOOOO... Don't just remove it. You need to add a few fees.

Reply:
Please follow my original PR. #125

Reply:
Done.